app/javascript/crayons/Indicator/__stories__/indicator.html.stories.jsx

```diff
@@ -30,12 +30,6 @@ GreyWithNumber.story = {
   name: 'grey with number',
 };
 
-export const GreyBullet = () => (
```

Comment:
These are the bullet type of indicators that we haven't been using anywhere soooo ✂️✂️✂️

Reply:
✂️ 🚀